tests/functions/test_transform_column.py

```diff
@@ -4,18 +4,26 @@
 
 
 @pytest.mark.functions
-def test_transform_column(dataframe):
-    # replacing the data of the original column
-
-    df = dataframe.transform_column("a", np.log10)
+@pytest.mark.parametrize("elementwise", [True, False])
+def test_transform_column(dataframe, elementwise):
+    """
+    Test replacing data of the original column.
+
+    The function that is used for testing here
+    must be able to operate elementwise
+    and on a single pandas Series.
+    We use np.log10 as an example
+    """
+
+    df = dataframe.transform_column("a", np.log10, elementwise=elementwise)
```

Comment:
A check to make sure that when `not elementwise`, an entire column is indeed passed:

```python
if not elementwise:
    def _test_is_series(series):
        assert isinstance(series, pd.Series)
        return series
    dataframe.transform_column("a", _test_is_series, elementwise=elementwise)
    
```

Reply:
Cool beans, thanks for the test suggestion!